src/plugins/intel_gpu/src/kernel_selector/cl_kernels/pooling_gpu_b_fs_zyx_fsv16_imad.cl

```diff
@@ -43,187 +43,6 @@ inline ACCUMULATOR_TYPE FUNC(apply_pooling)(ACCUMULATOR_TYPE tmp, ACCUMULATOR_TY
 #endif
 }
 
-#if GLOBAL_POOLING
```

Comment:
I believe this code can be used as GLOBAL_POOLING value is defined by func below:
```cpp
bool PoolingKernelGPU_b_fs_zyx_fsv16_imad::IsGlobalPooling(const pooling_params& params) const {
    return params.outputs[0].X().v == 1 && params.outputs[0].Y().v == 1 && params.outputs[0].Z().v == 1;
}
```
So removal may lead to perf drop

Reply:
Reverted

Reply:
I still see this change